Frontend/android-native/mobility-customer/src/main/res/values-hi/strings.xml

```diff
@@ -50,4 +50,6 @@
     <string name="driver_receipt_downloaded">Driver Receipt Downloaded!!!</string>
     <string name="driver_receipt_for_your_ride_is_downloaded">Driver Receipt for your ride is downloaded!!!</string>
     <string name="invoice_for_your_ride_is_downloaded">Invoice for your ride is downloaded!!!</string>
+    <string name="no_additional_fare_tips_and_waiting_charges">"कोई अतिरिक्त किराया नहीं। कोई सुझाव नहीं। कोई प्रतीक्षा शुल्क नहीं।"</string>
+    <string name="third_party_booking">"तीसरे पक्ष की बुकिंग"</string>
```

Comment:
These strings are not used in mobility customer, Please remove from mobility-customer